### ISSUE TITLE
JDK11 throws the RuntimeException, but JDK8 executes normally

### ISSUE THREAD

**Issue by JavaTailor** [eclipse-openj9/openj9#16202]
## Affected versions

We found a test case with execution problems. To facilitate analysis, we simplified the test case and the simplified class file can ben found at Attachment.

Windows 10:

```
Microsoft Windows 10 Professional
10.0.19044 Build 19044
AMD Ryzen 5 5600G with Radeon Graphics 3.90 GHz
Memory 32 GB
```

Java -version output under Windows 10

```
openjdk version "1.8.0_332"
IBM Semeru Runtime Open Edition (build 1.8.0_332-b09)
Eclipse OpenJ9 VM (build openj9-0.32.0, JRE 1.8.0 Windows 10 amd64-64-Bit Compressed References 20220422_375 (JIT enabled, AOT enabled)
OpenJ9   - 9a84ec34e
OMR      - ab24b6666
JCL      - 0b8b8af39a based on jdk8u332-b09)
```

```
openjdk version "11.0.15" 2022-04-19
IBM Semeru Runtime Open Edition 11.0.15.0 (build 11.0.15+10)
Eclipse OpenJ9 VM 11.0.15.0 (build openj9-0.32.0, JRE 11 Windows 10 amd64-64-Bit Compressed References 20220422_352 (JIT enabled, AOT enabled)
OpenJ9   - 9a84ec34e
OMR      - ab24b6666
JCL      - b7b5b42ea6 based on jdk-11.0.15+10)
```

## Problem summary
This is a repeat submission of #16078 and would like to explain the details related to the issue. We also submit relevant issue reports to the hotspot developers. They believe that the reported issue was confirmed to exist before JDK 17.0.4.1 but not observed in later versions (18,19,20). Since this problem has not been observed in later releases, the initial fix will be ported back to JDK 17 and 11.

When we execute TestCase22 using jdk8 and jdk11, we get the following output:

```
TestCase22 {
  Test4486735 Passed
} Passed

TestCase22 {
  Test4486735 {
    Uncaught exception thrown in test method Test4486735
Exception in thread "main" java.lang.RuntimeException: Uncaught exception thrown in test method Test4486735
        at TestCase22_interface.errImpl(IntlTest.jasm)
        at TestCase22_interface.errln(IntlTest.jasm)
        at TestCase22_interface.run(IntlTest.jasm)
        at TestCase22.main(DateFormatTest.jasm)
```

## Attachment
[TestCase22.zip](https://github.com/eclipse-openj9/openj9/files/9869680/TestCase22.zip)

**Comment by pshipton** [comment]
What is the Htospot issue?

**Comment by keithc-ca** [comment]
The (unexpected) exception I see using Java 11 is
```
java.text.ParseException: Unparseable date: "miércoles, 26 de octubre de 2022, 17:53:00 hora de verano británica"
```

**Comment by JavaTailor** [comment]
> Clarified my previous question: "What is the OpenJDK issue link?"

The URL of the report we submitted：https://bugs.openjdk.org/browse/JDK-8294889

**Comment by pshipton** [comment]
Seems it will be fixed in 11.0.20 via https://bugs.openjdk.org/browse/JDK-8305372

**Comment by pshipton** [comment]
I see the fix in
jdk11 https://github.com/ibmruntimes/openj9-openjdk-jdk11/commit/8c2305677
jdk17 https://github.com/ibmruntimes/openj9-openjdk-jdk17/commit/1854496b8
jdk20 https://github.com/ibmruntimes/openj9-openjdk-jdk20/commit/e89737148
